<launch>
<node name="interaction_generator" pkg="cr_week8_test" type="interaction_generator.py" output="screen">
</node>

<node name="perception_filter" pkg="cr_week8_test" type="perception_filter.py" output="screen">
</node>

<node name="robot_expression_prediction" pkg="cr_week8_test" type="robot_expression_prediction.py" output="screen">
</node>

<node name="robot_controller" pkg="cr_week8_test" type="robot_controller.py" output="screen">
</node>

<node name="rqt_graph_GUI" pkg="rqt_graph" type="rqt_graph">
</node>
</launch>
